src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.groovy

```diff
@@ -44,7 +45,11 @@ class CreateManifestTask extends DefaultTask {
 
     @Input
     public String getProjectVersion() {
-        return String.valueOf(project.version)
+        def stringVersion = String.valueOf(project.version)
+        if (!SlsProductVersions.isValidVersion(stringVersion)) {
```

Comment:
Probably worth a noisy-ish non-blocking warning if this isn't also an orderable version.